### PR TITLE
add ssl errors

### PR DESCRIPTION
**Pull request by amchang**
Need to add this options as this is used for a lot of localhost ssl issues

### CODE REVIEW

**Comment by amchang** [comment]
Also if good, can you publish this up? I need this for the various changes I'll be making in mochify.js

**Comment by indexzero** [comment]
Just a side note it might be worthwhile to be consistent with other naming conventions for this:

- `{ "rejectUnauthorized": false }` in `node` (`https` module): https://nodejs.org/api/https.html#https_https_request_options_callback
- ``{ "strictSSL": false }` in `request`: https://github.com/request/request#requestoptions-callback

File: README.md [hunk 1]
```diff
@@ -30,6 +30,9 @@ Options:
     --web-security <bool>  Enables PhantomJS web security and forbids
                            cross-domain XHR (default is true)
     --brout                Assume brout is part of the JS
+    --ignore-ssl-errors <bool>  Tell PhantomJS web security and forbids
+                                cross-domain XHR (default is false)
```
Comment (mantoni) [inline README.md]: This doesn't seem like the right description. Is this a copy-paste from the `--web-security` option?

**Comment by mantoni** [comment]
@indexzero I think the option name is fine. It's just forwarded to PhantomJS and that is what it's called over there. Why should it have a different name?

**Comment by indexzero** [comment]
@mantoni didn't catch that reading the diff. That makes sense.

File: README.md (hunk 1 above)
Comment (amchang) [inline README.md]: oops my mistake, I will fix that

**Comment by mantoni** [comment]
@amchang Oh, just saw that the file `test/ignore-ssl-errors.js` is missing.

**Comment by amchang** [comment]
I'll make another PR must of forgot to commit that in

**Comment by mantoni** [comment]
Released in `1.2.0`.

**Pull request merged**